Runtime/CSharp/Bitcraft.StateMachine/EventArgs.cs

```diff
@@ -80,10 +99,12 @@ public class StateChangedEventArgs : EventArgs
         /// <summary>
         /// Initializes the StateChangedEventArgs instance.
         /// </summary>
+        /// <param name="triggeringAction">Action token of the action that triggered the transition.</param>
         /// <param name="oldState">Old state.</param>
         /// <param name="newState">New state.</param>
```

Comment:
Out of scope of this PR, but still do it please: those two deserve better description. 🙇